concourse/pipelines/deploy.yml

```diff
@@ -291,9 +309,27 @@ jobs:
               -auto-approve
 
             terraform output -json > "$root_dir/govuk-terraform-outputs/govuk-terraform-outputs.json"
+
+            for app in content-store; do
+              terraform output -json "${app}" > "$root_dir/updated-${app}-terraform-outputs/${app}.json"
```

Comment:
I wonder if a new task to do the app-based terraform output comparison should be created? Because how will this code be scaled for the dozens of apps that we have? Maybe a parameterizable task would be more appropriate?

Reply:
The intention is just to keep adding app names to the for loop. So it will end up like:

```bash
for app in content-store frontend publisher publshing-api router router-api etc...; do
   terraform output -json "${app}" ...
```

Reply:
I agree that's a bit disgusting though.

Reply:
I've changed the for loop to a function in 0c3c0dd - hopefully that's a little bit less confusing?